app-misc/datovka/datovka-4.23.7.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="GUI to access the Czech data box e-government system"
+HOMEPAGE="https://www.datovka.cz/"
+SRC_URI="https://secure.nic.cz/files/datove_schranky/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# minimum Qt version required
+QT_PV="5.14.0:5"
+
+RDEPEND="
+	>=dev-libs/openssl-1.0.2:=
+	dev-libs/quazip:=[qt5(+)]
+	>=dev-qt/qtcore-${QT_PV}
+	>=dev-qt/qtgui-${QT_PV}
+	>=dev-qt/qtnetwork-${QT_PV}[ssl]
+	>=dev-qt/qtprintsupport-${QT_PV}
+	>=dev-qt/qtsql-${QT_PV}[sqlite]
+	>=dev-qt/qtsvg-${QT_PV}
+	>=dev-qt/qtwebsockets-${QT_PV}
+	>=dev-qt/qtwidgets-${QT_PV}
+	>=app-misc/libdatovka-0.6.0
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	>=dev-qt/linguist-tools-${QT_PV}
+	virtual/pkgconfig
+"
+
+DOCS=( ChangeLog README )
```

Comment:
i think you could remove the `DOCS` variable here. Per default `src_install` already installs a certain set of doc files, which should include these files.